### PR TITLE
Add pyqmix

### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,27 +1,674 @@
-Copyright (c) 2015-2018, conda-forge
```
Comment (synapticarbors) [inline LICENSE.txt]: Please do not modify this license file

File: recipes/pyqmix/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/pyqmix/meta.yaml]: Please remove all comments

File: recipes/pyqmix/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: GPL-3.0
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  # license_family: MIT
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/pyqmix/meta.yaml]: I think this should just be `LICENSE`

File: LICENSE.txt (hunk 1 above)
Comment (hoechenberger) [reply LICENSE.txt]: Thanks for your feedback, @synapticarbors. I had modified the license file to match the license of the Python package I intend to build for conda-forge. Is this not what the license file is supposed to reflect? At least that's how I understood the documentation.

File: recipes/pyqmix/meta.yaml (hunk 3 above)
Comment (hoechenberger) [reply recipes/pyqmix/meta.yaml]: `LICENSE.txt` is the default name of the license file in https://github.com/conda-forge/staged-recipes (from which I forked). Shall I rename the file anyway?

File: recipes/pyqmix/meta.yaml (hunk 2 above)
Comment (hoechenberger) [reply recipes/pyqmix/meta.yaml]: This has been resolved in the latest commit.

**Comment by hoechenberger** [comment]
@synapticarbors I think this should be good now.

**Comment by synapticarbors** [comment]
Looks generally good, but one final question -- why did you switch from the `script` line given in the `example/meta.yaml` (https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33)

**Pull request merged**